errors/multierror.go

```diff
@@ -18,12 +18,24 @@ func NewMultiError(errs ...error) error {
 	}
 
 	if len(nilsRemoved) == 0 {
-		return nil
+		return MultiError{}
```

Comment:
My guess is this change will break things, because now we're always returning errors...